elements/lisk-p2p/package.json

```diff
@@ -65,6 +65,7 @@
 		"@types/jsonwebtoken": "8.3.0",
 		"@types/lodash.clonedeep": "4.5.0",
 		"@types/lodash.shuffle": "4.2.6",
+		"@types/lodash.uniq": "4.5.0",
```

Comment:
I just realized that we don't have `clonedeep` and `uniq` installed, but seems like using it